www/_posts/2020-06-01-cordova-ios-release-6.0.0.md

```diff
@@ -0,0 +1,108 @@
+---
+layout: post
+author:
+    name: Bryan Ellis
+title:  "Cordova iOS 6.0.0 Released!"
+categories: announcements
+tags: news releases
+---
+
+We are happy to announce that we have just released `Cordova iOS 6.0.0`! This is Cordova's official platform for building iOS mobile applications.
+
+* [cordova-ios@6.0.0](https://www.npmjs.com/package/cordova-ios)
+
+## Release Highlights
+
+**To upgrade:**
+
+```bash
+cordova platform remove ios
+cordova platform add ios@6.0.0
+```
+
+The most notable changes in this major release are:
+
+* Bumped minimum iOS version to 11.0
+* Added Xcode 11 compatibility
+* Moved `WKWebView` support into **Cordova-iOS** and removed `UIWebView` code
```

Comment:
We have super inconsistent rules about what/when to bold things in these posts. I think it's better if Cordova-iOS is not bolded here.